salt/exceptions.py

```diff
@@ -344,6 +344,13 @@ class EauthAuthenticationError(SaltException):
     """
 
 
+class EauthAuthorizationError(SaltException):
+    """
+    Thrown when eauth user was authenticated, but they were not authorized
+    to perform the requested action.
+    """
+
+
```

Comment:
This was added when I realized that Authentication != Authorization. The previous failure is failure to login, at this point (generally speaking) the external auth user has been authenticated, they're just not **Authorized** to perform an action, and thus :point_up: this exception.